velox/expression/tests/ArrayProxyTest.cpp

```diff
@@ -278,5 +278,43 @@ TEST_F(ArrayProxyTest, testTimeStamp) {
   assertEqualVectors(result, makeNullableArrayVector(expected));
 }
 
+TEST_F(ArrayProxyTest, testString) {
+  auto result =
+      prepareResult(std::make_shared<ArrayType>(ArrayType(VARCHAR())));
+
+  exec::VectorWriter<ArrayProxyT<Varchar>> writer;
+  writer.init(*result.get()->as<ArrayVector>());
+  writer.setOffset(0);
+  auto& arrayProxy = writer.current();
+  // General interface only is allowed for arrays of strings.
+  {
+    auto& string = arrayProxy.add_item();
```

Comment:
a `push_back` that takes a `std::string_view` doesn't seem like it would be too bad.

Reply:
it would do the string copy i assume, the users might then not used write strings in the most efficient way